src/main/java/com/cleanroommc/groovyscript/compat/mods/additionalenchantedminer/WorkbenchPlusRecipe.java

```diff
@@ -0,0 +1,69 @@
+package com.cleanroommc.groovyscript.compat.mods.additionalenchantedminer;
+
+import com.cleanroommc.groovyscript.api.IIngredient;
+import com.yogpc.qp.recipe.WorkbenchRecipe;
+import com.yogpc.qp.tile.ItemDamage;
+import com.yogpc.qp.utils.IngredientWithCount;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import scala.Option;
+import scala.collection.JavaConversions;
+import scala.collection.Map;
+import scala.collection.Seq;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.yogpc.qp.recipe.WorkbenchRecipe.addIngredientRecipe;
+
+public class WorkbenchPlusRecipe {
```

Comment:
what is the point of this class? what purpose does it serve that it couldnt be replaced by some code in the `register()` method?

Reply:
Now that I think about it, I realize that I don't need WorkbenchPlusRecipe, I'll fix it.

Reply:
nothing changed here.